<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetronomeConfigBase</class>
 <widget class="QDialog" name="MetronomeConfigBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusE: Metronome Config</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="GroupBox1">
       <property name="title">
        <string>Metronome</string>
       </property>
       <layout class="QGridLayout">
        <property name="margin">
         <number>11</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QCheckBox" name="audioBeep">
              <property name="text">
               <string>Audio Beep</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="audioBeepRoutesButton">
              <property name="text">
               <string>Choose outputs...</string>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QSlider" name="volumeSlider">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="pageStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="volumeLabel">
              <property name="text">
               <string>50</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel1">
              <property name="text">
               <string>% Audio volume</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="midiClick">
            <property name="text">
             <string>MIDI Click</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QGridLayout">
              <property name="margin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="4" column="1">
               <widget class="QLabel" name="TextLabel5">
                <property name="text">
                 <string>Midi Channel</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="TextLabel1">
                <property name="text">
                 <string>Measure Note</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QSpinBox" name="midiChannel">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>16</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="TextLabel2">
                <property name="text">
                 <string>Measure Velocity</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="TextLabel4">
                <property name="text">
                 <string>Beat Velocity</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QSpinBox" name="midiPort">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>16</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="TextLabel3">
                <property name="text">
                 <string>Beat Note</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QSpinBox" name="beatNote"/>
              </item>
              <item row="1" column="0">
               <widget class="QSpinBox" name="measureVelocity">
                <property name="maximum">
                 <number>127</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QSpinBox" name="measureNote"/>
              </item>
              <item row="5" column="1">
               <widget class="QLabel" name="TextLabel6">
                <property name="text">
                 <string>Midi Port</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QSpinBox" name="beatVelocity">
                <property name="maximum">
                 <number>127</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="Spacer10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="GroupBox2">
       <property name="title">
        <string>Precount</string>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>11</number>
        </property>
        <item>
         <widget class="QCheckBox" name="precountEnable">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>enable</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSpinBox" name="precountBars">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel7">
            <property name="text">
             <string>Bars</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="precountFromMastertrack">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>From Mastertrack</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSpinBox" name="precountSigZ">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>32</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel8">
            <property name="text">
             <string>/</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="precountSigN">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>64</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel9">
            <property name="text">
             <string>Signature</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="precountPrerecord">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Prerecord</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="precountPreroll">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Preroll</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="textLabel3">
         <property name="text">
          <string>Hint: Enable metronome in Transportpanel</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Horizontal Spacing2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>8</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonApply">
         <property name="text">
          <string>&amp;Apply</string>
         </property>
         <property name="shortcut">
          <string>Alt+A</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOk">
         <property name="text">
          <string>&amp;OK</string>
         </property>
         <property name="shortcut">
          <string>Alt+O</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCancel">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
         <property name="shortcut">
          <string>Alt+C</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>audioBeep</tabstop>
  <tabstop>audioBeepRoutesButton</tabstop>
  <tabstop>midiClick</tabstop>
  <tabstop>measureNote</tabstop>
  <tabstop>measureVelocity</tabstop>
  <tabstop>beatNote</tabstop>
  <tabstop>beatVelocity</tabstop>
  <tabstop>midiChannel</tabstop>
  <tabstop>midiPort</tabstop>
  <tabstop>precountEnable</tabstop>
  <tabstop>precountBars</tabstop>
  <tabstop>precountFromMastertrack</tabstop>
  <tabstop>precountSigZ</tabstop>
  <tabstop>precountPrerecord</tabstop>
  <tabstop>precountPreroll</tabstop>
  <tabstop>precountSigN</tabstop>
  <tabstop>buttonApply</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>MetronomeConfigBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>MetronomeConfigBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>volumeLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
